### PR TITLE
Feature/python bindings

### PR DESCRIPTION
**Pull request by asadchev**

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -14,8 +14,10 @@ jobs:
         include:
           - os: ubuntu-20.04
             cxx: /usr/bin/g++-9
+            libint2-python: ON
           - os: macos-latest
             cxx: clang++
+            libint2-python: OFF
```
Comment (evaleev) [inline .github/workflows/cmake.yml:20]: @asadchev why not test python on macOS?

File: include/libint2/solidharmonics.h [hunk 2]
```diff
@@ -215,6 +160,62 @@ namespace libint2 {
           return result;
         }
 
+      private:
```
Comment (evaleev) [inline include/libint2/solidharmonics.h:163]: just spaces->tabs or vice versa?

File: .github/workflows/cmake.yml (hunk 1 above)
Comment (asadchev) [reply .github/workflows/cmake.yml:20]: i thought to have some workflow that test with python off.  you want treat all?

**Comment by evaleev** [comment]
@asadchev are you going to squash and re-open?

**Comment by asadchev** [comment]
yes, had recreate branch, had a git issue

**Pull request closed without merging**